app/views/pages/get-started/build-your-first-app/sending-email-notifications.liquid

```diff
@@ -0,0 +1,215 @@
+) {
+  email_send(
+    template: { name: "item/created" }
+    data: $data
+  ) {
+    is_scheduled_to_send
+    errors { message }
+  }
+}
+```
+
+We define a new _mutation_ – which is a GraphQL name for any operation that modify or create data – in the 1st line. The name of the mutation is just there for our convenience.
+
+In the 2nd line we are defining a variable called `data`. In GraphQL variable name is prefixed with an dollar sign (`$`). This will be the variable with all of the information we want to send in the email, so to handle all of that data we need it to be a _hash_ (also called an _object_ in other programming languages). The exclamation mark (`!`) in the end means that this variable is required for the mutation to run.
+
+Then, in the line 4th, we are using a function that will send the email using the Liquid template that we created in the previous steps of this tutorial (line 5th). In line 6th we are passing the `data` variable to this template. If you’ll check the `app/emails/item/created.liquid` file you will notice that we are using this `data` hash to output the title in the email’s content.
+
+Starting from the line 8 we are defining what would we like to get returned after the mutation will do it’s thing. The `is_scheduled_to_send` is a boolean that will tell us if everything went right (when `true`) and the email is added to the queue and will be send. Alternatively, if something went wrong, we will have the error message returned, as defined in the 9th line.
+
+
+## Trigger sending the email
+
+When we have [enabled emails on staging](#enabling-emails-on-the-staging-instance), [created the HTML](#creating-a-layout-for-the-emails) for the emails and [prepared the mutation](#using-graphql-to-send-an-email), the only thing left to do is to run the mutation when needed using Liquid.
+
+{% raw %}
+```liquid
+{% liquid
+graphql email = 'item/email/created', data: <hash>
+%}
+```
+{% endraw %}
+
+This simple line runs the mutation saved in `app/graphql/item/email/created.graphql`. We don’t need to pass the whole path to the file as the platform knows where to look for GraphQL files. We are also passing to it a single variable called `data` that we will use to assign the hash with the information needed for sending the email.
+
+If you are following this tutorial and building the _To Do_ app, then you probably would like to send the email after a new item has been added to the list.
+
+Since we have a separate page that handles this (`app\views\pages\item\create.liquid`), we can just plug into the right place and trigger the GraphQL mutation after the new item was successfully saved in the database:
+
+#### /app/views/pages/item/create.liquid
+
+{% raw %}
+```liquid
+---
+method: post
+---
+
+
+{% liquid
+	graphql result = 'item/create', title: context.params.title
+
+	if result.record.id
++		graphql email = 'item/email/created', data: context.params
+		redirect_to '/'
+	else
+		echo 'Something went wrong'
```

Comment:
In general we'd like to suggest treating Page as a controller and not to render anything directly, so it would be great if we could have some kind of `error` partial which does it, what do you think? something like:
`render "errors/something_went_wrong", errors: result.errors`

Reply:
Yup, I think the concept of not using pages for presentation layers should be introduced later on, not in the beginner tutorial, we have a lot of information here either way. I did planned a section called 'Advanced concepts' where we could do something like 'Separating data handling and business logic from the UI'.